### PR TITLE
issue-1588: update CSI driver README and deployment manifests

### PR DESCRIPTION
**Pull request by antonmyagkov**
issue: #1588 

Changes:

1. Remove links to Nebius internal infrastructure( container registry, nbs pull secrets)
2. build local CSI Driver without publishing it in remote container registry
3.  fix nbs-csi-driver parameters and mountPath
4.  remove nbs-server manifests(Run nbsd-lightweight instead of it)
5. run minikube with docker driver to avoid "blkUtil.AttachFileDevice failed" error: https://github.com/kubernetes/minikube/issues/8284
6. Add instruction how to create volume manually via blockstore-client

### CODE REVIEW

File: cloud/blockstore/tools/csi_driver/README.md [hunk 1]
```diff
@@ -50,38 +50,62 @@ minikube stop
...
-# Pull docker image with csi-driver on the node
+# Build csi-driver
+```bash
+./ya make -r cloud/blockstore/tools/csi_driver
+
+```
+
+
+# Build docker container with csi-driver
+# Need to replace csi_driver symlink to csi_driver binary before building docker image
```
Comment (aikuchin) [inline cloud/blockstore/tools/csi_driver/README.md:61]: Steps to replace binary
```
DRIVER_PATH=$(readlink -f "./cloud/blockstore/tools/csi_driver/cmd/nbs-csi-driver/nbs-csi-driver")
rm ./cloud/blockstore/tools/csi_driver/cmd/nbs-csi-driver/nbs-csi-driver
cp ${DRIVER_PATH} ./cloud/blockstore/tools/csi_driver/cmd/nbs-csi-driver/nbs-csi-driver
```

Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/README.md]: Done

File: cloud/blockstore/tools/csi_driver/README.md [hunk 2]
```diff
@@ -50,38 +52,60 @@ minikube stop
...
+kubectl apply -f ./deploy/manifests/6-csi-daemonset.yaml
 ```
 
-# copy nbsd and blockstore-nbd to minikube and run them
+# copy nbsd-lightweight, blockstore-nbd and blockstore-client to minikube and run them
 ```bash
-sudo ./nbsd-lightweight --service local --server-file ./server/server.txt --verbose
-sudo ./blockstore-nbd --device-mode endpoint --disk-id my-disk --access-mode rw --mount-mode local --connect-device /dev/nbd0 --listen-path /tmp/nbd.sock
+./ya make -r cloud/blockstore/apps/server_lightweight
+minikube cp cloud/blockstore/apps/server_lightweight/libiconv.so /home/libiconv.so
```
Comment (dvrazumov) [inline cloud/blockstore/tools/csi_driver/README.md:92]: You may build with `-DFORCE_STATIC_LINKING=yes` to avoid libraries copying

Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/README.md]: Done. Thanks!

File: cloud/blockstore/tools/csi_driver/deploy/server.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    DataHost: "localhost"
+    DataPort: 9767
+    StrictContractValidation: true
+    UnixSocketPath: "/run/nbsd/grpc.sock"
+    ThreadsCount: 4
+    VhostEnabled: true
+    VhostThreadsCount: 4
+    NodeType: "nbs"
+    EndpointStorageType: ENDPOINT_STORAGE_FILE
+    EndpointStorageDir: "/Berkanavt/nbs-server/endpoints"
```
[the remark this answers is not in the document]
Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/deploy/server.txt]: Done

**Review by dvrazumov — APPROVED** [review]

**Review by drbasic — APPROVED** [review]

**Pull request merged**